src/convert.ts

```diff
@@ -16,17 +17,34 @@ export const defaultArgs = [
 
 const INPUT_PATH = '/opt/lo.tar.br';
 const OUTPUT_PATH = '/tmp/instdir/program/soffice.bin';
+const UNOPKG_OUTPUT_PATH = '/tmp/instdir/program/unopkg.bin';
 
 /**
  * Converts a file in /tmp to the desired file format
  * @param {String} filename Name of the file to convert located in /tmp directory
  * @param {String} format File format to convert incoming file to
+ * @param {Array} extensions List of LibreOffice extension paths (.oxt files)
+ * @param {Boolean} shouldThrowOnExtensionFail Throw exceptions if extension was not loaded
  * @return {Promise<String>} Absolute path to the converted file
  */
-export async function convertTo(filename: string, format: string): Promise<string> {
+export async function convertTo(
+  filename: string,
+  format: string,
+  extensions: string[] = [],
+  shouldThrowOnExtensionFail = true
+): Promise<string> {
   let logs;
   cleanupTempFiles();
   await unpack({inputPath: INPUT_PATH});
+
+  if (extensions.length) {
+    const enabledExtensions = execSync(`${UNOPKG_OUTPUT_PATH} list --shared`);
```

Comment:
```suggestion
    const enabledExtensions = execSync(`${UNOPKG_OUTPUT_PATH} list --shared`).toString();
```

Because below you do `enabledExtensions.includes` which won't work on a `Buffer` type